File: Sandtree/Game/Sandtree/Assets/Code/ContextSpotScript.js
#pragma strict

var camPosV3	: Vector3;
var camRotV3	: Quaternion;
var newLookPoint: Vector3;

var	posObj	: Transform;
var targObj	: Transform;

function Start ()
{
	posObj. gameObject.GetComponent (MeshRenderer).enabled	= false;
	targObj.gameObject.GetComponent (MeshRenderer).enabled	= false;
	newLookPoint	= targObj.position;
	var relative	= targObj.position - posObj.position;
	camPosV3	= posObj.position;
	camRotV3	= Quaternion.LookRotation (relative);
}

File: Sandtree/Game/Sandtree/Assets/Code/ContextSpotScript.cs
using UnityEngine;
using System.Collections;

public class ContextSpotScript : MonoBehaviour {

	public Vector3 camPosV3;
	public Quaternion camRotV3;
	public Vector3 newLookPoint;

	public Transform posObj;
	public Transform targObj;

	void Start ()
	{
		posObj.gameObject.GetComponent <MeshRenderer>().enabled = false;
		targObj.gameObject.GetComponent <MeshRenderer>().enabled	= false;
		newLookPoint	= targObj.position;
		var relative	= targObj.position - posObj.position;
		camPosV3	= posObj.position;
		camRotV3	= Quaternion.LookRotation (relative);
	}
}
